Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -2784,15 +2758,12 @@ private async Task ScanForTunerDeviceChanges(ITunerHost host, CancellationToken
             {
                 var configuredDevice = configuredDevices.FirstOrDefault(i => string.Equals(i.DeviceId, device.DeviceId, StringComparison.OrdinalIgnoreCase));
 
-                if (configuredDevice != null)
+                if (configuredDevice != null && !string.Equals(device.Url, configuredDevice.Url, StringComparison.OrdinalIgnoreCase))
                 {
-                    if (!string.Equals(device.Url, configuredDevice.Url, StringComparison.OrdinalIgnoreCase))
-                    {
-                        _logger.LogInformation("Tuner url has changed from {0} to {1}", configuredDevice.Url, device.Url);
+                    _logger.LogInformation("Tuner url has changed from {0} to {1}", configuredDevice.Url, device.Url);
```

Comment:
While you are cleaning you could put a useful name inside the braces instead of 0 and 1